src/main/java/org/folio/dataexp/service/export/strategies/ld/LinkedDataProvider.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.dataexp.service.export.strategies.ld;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.folio.dataexp.service.QueryService;
+import org.springframework.stereotype.Component;
+
+/**
+ * Retrieve Linked Data resources.
+ */
+@Component
+@RequiredArgsConstructor
+public class LinkedDataProvider {
+
+  private final QueryService queryService;
+
+  private static final UUID LINKED_DATA_RESOURCE =
+      UUID.fromString("84451307-215c-4021-a46d-b9dcfada7439");
+  private static final String GRAPH_FIELD = "resource_subgraph";
+  private static final List<String> RESOURCE_FIELDS = List.of("inventory_id", GRAPH_FIELD);
```

Comment:
Can this be saved as constant similar to GRAPH_FIELD - INVENTORY_ID = "inventory_id"?